views/index.jade

```diff
@@ -5,21 +5,24 @@ block content
     #top
       #logo.text-center
         img.logo(src='./smokeAlarmPortal_files/redcross-logo-white-letters.png', alt='American Red Cross')
+        if origin == '/311'
+          img.img-square(src='./images/311-chicago.png', alt='311 Chicago')
       #safety-message.row.text-center(role='complementary')
         p.col-xs-12.bigger-centered
           | #{__('Smoke alarms save lives.  Installing a smoke alarm is the first step to keeping your family safe.')}
-    #language
+      #language
       .row
         p.col-xs-12.bigger-centered
             a.language-toggle(data-lang='en') English
             | &nbsp;|&nbsp;
             a.language-toggle(data-lang='es') Español
     #main.row(role='main')
       #left.col-md-3
-        p
-          | #{__('Thank you for your interest in having FREE smoke alarms installed in your home.')} #{__('To request a FREE installation, please complete all fields in the form below and click “Submit.”')}
-        p
-          | #{__('After submitting your request, you will be contacted by an American Red Cross representative with information on installation availability in your area.')}
+        if origin != '/311'
```

Comment:
Yup.  These conditionals all make sense.  Later, if/when we add more city 311 routes, then we'll need some kind of global conditional indicating "This is a 311 route for some city", but that won't be identical to the `origin`, excuse me I mean the `source` :-) that we're using.  Nothing needs to be changed now; just noting that a future generalization is likely.  If you think of a strategic place for a comment to that effect -- a place where someone making such a generalization change would encounter the comment early -- that might be good to put in.